moc_openstack_api_reporting.py

```diff
@@ -734,7 +734,7 @@ def main():
 
     vol_keys = list(cluster_state["vol_timestamps"].keys())
     for key in vol_keys:
-        if cluster_state["vol_timestamps"][key]["updated"] == 0:
+        if "updated" in list(cluster_state["vol_timestamps"][key].keys()) and cluster_state["vol_timestamps"][key]["updated"] == 0:
```

Comment:
A better way to write this is `if cluster_state["vol_timestamps"][key].get("updated") == 0:`

The get method returns `None` if the dictionary doesn't contain the key, and None is different from 0.

Reply:
sure